src/Screens/AddContact.screen.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import { View } from 'react-native';
 
 import Button from '../UIKit/Button';
+import AddContact from '../Components/AddContacts';
 
 import * as Routes from '../Constants/navigationRouts';
 
-const AddContactScreen = () => <View />;
+const AddContactScreen = () => <AddContact />;
```

Comment:
let instead of returning AddContact component move AddContact data into this screen